proposals/1/loadSet.md

```diff
@@ -0,0 +1,129 @@
+# Load Set
+
+## Current situation
+
+-  The hardcoded path for mod installation or discovery is `assets/mods/`. The mod must exist there to be found. 
```

Comment:
This is not correct. https://github.com/CCDirectLink/CLS/blob/master/standards/standardized-mod-format.md#mod-format states that any mod directory can be chosen. Instead I'd suggest something along the lines of "there is only one possible mod folder"

Reply:
If you read it more closely, it does not suggest anything about where the mod is located. It is referring to the actual mod folder name is irrelevant. I do agree that it could be worded better.